src/libs/ReportUtils.ts

```diff
@@ -6071,7 +6080,9 @@ function buildOptimisticMovedReportAction(fromPolicyID: string | undefined, toPo
 
     const movedActionMessage = [
         {
-            html: `moved the report to the <a href='${CONST.NEW_EXPENSIFY_URL}r/${newParentReportID}' target='_blank' rel='noreferrer noopener'>${policyName}</a> workspace`,
+            html: isIouReport
+                ? `moved the <a href='${CONST.NEW_EXPENSIFY_URL}r/${reportID}/${reportActionID}' target='_blank' rel='noreferrer noopener'>report</a> to the ${policyName} workspace`
```

Comment:
Let's be consistent with the backend copy:

```suggestion
                ? `moved this <a href='${CONST.NEW_EXPENSIFY_URL}r/${reportID}/${reportActionID}' target='_blank' rel='noreferrer noopener'>report</a> to the ${policyName} workspace`
```